sequencing/ATACSeq/subScripts/compareBamWithGenotypes.sh

```diff
@@ -43,47 +58,58 @@ then
-
+    java -jar $EBROOTPICARD/picard.jar  MarkDuplicates INPUT=${ALIGNED_DIR}/${bamfile} OUTPUT=${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup.bam METRICS_FILE=${ALIGNED_DIR}/baseRecalibrate/${sampleName}_metrics.txt     
 
     ## add read group
     java -jar $EBROOTPICARD/picard.jar AddOrReplaceReadGroups \
-           I=baseRecalibrate/${sampleName}_dedup.bam \
-           O=baseRecalibrate/${sampleName}_dedup_rglabelled.bam \
+           I=${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup.bam \
+           O=${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup_rglabelled.bam \
            RGID=${projectID} \
            RGLB=lib1 \
            RGPL=ILLUMINA \
            RGPU=unit1 \
            RGSM=${sampleName}
 
-    rm baseRecalibrate/${sampleName}_dedup.bam
+    rm ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup.bam
 
     ##index
-    samtools index baseRecalibrate/${sampleName}_dedup_rglabelled.bam
+    samtools index ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup_rglabelled.bam
 
     # recalibrate bases in bam files
     gatk BaseRecalibrator \
         -R ${GENOMEFASTA}/genome.fa \
-        -I baseRecalibrate/${sampleName}_dedup_rglabelled.bam \
+        -I ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup_rglabelled.bam \
         --known-sites ${KGREF}/1000G_omni2.5.hg38.vcf.gz \
-        -O baseRecalibrate/${sampleName}_recal_data.table
+        -O ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_recal_data.table
 
     gatk ApplyBQSR \
-       -R ${GENOMEFASTA}/genome.fa \
-       -I baseRecalibrate/${sampleName}_dedup_rglabelled.bam \
-       --bqsr-recal-file baseRecalibrate/${sampleName}_recal_data.table \
-       -O baseRecalibrate/${sampleName}_baserecal.bam
+      -R ${GENOMEFASTA}/genome.fa \
+       -I ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup_rglabelled.bam \
+       --bqsr-recal-file ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_recal_data.table \
+       -O ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_baserecal.bam
        
-    rm baseRecalibrate/${sampleName}_dedup_rglabelled.bam*
-	rm baseRecalibrate/${sampleName}_recal_data.table
-
-    mkdir -p genotypeConcordance
-
-    ${VERIFYBAMID} --vcf ${GENODIR}/hg38/allchr_filt_rsq_maf.vcf.gz --bam baseRecalibrate/${sampleName}_baserecal.bam --out genotypeConcordance/${sampleName} --ignoreRG --smID ${vcfid} --self
+    rm ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup_rglabelled.bam*
+	  rm ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_recal_data.table
 
+    mkdir -p ${ALIGNED_DIR}/genotypeConcordance
+    
+    ${VERIFYBAMID} --vcf ${SNP} --bam ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_baserecal.bam --out ${ALIGNED_DIR}/genotypeConcordance/${sampleName} --ignoreRG --smID ${vcfid} --self
 else
 
     echo "No genotype data available"
+    echo $sampleName, >> $META_DIR/noVCFfound.txt
```

Comment:
Why is there a comma at the end of each of these lines in the `noVCFfound.txt` file? I guess this file is never being read anywhere (not yet at least), so it doesn't really matter. Just curious